src/components/Profile.js

```diff
@@ -57,35 +54,26 @@ class Profile extends Component<Props> {
       && !this.props.trustedByUids.has(this.props.authMemberDoc.id);
   }
 
-  isOwnProfile(uid) {
-    return this.props.authMemberDoc && this.props.authMemberDoc.uid === uid;
+  isOwnProfile() {
```

Comment:
It would probably be simplest if we just had a selector at the Redux level that computes this based on the state of the application—we can hook up `react-router` to also sync its state in the Redux store, and then remove this aspect of business logic from the component level. But that can happen in a later PR.

Reply:
does this mean that the current route would be managed by changes to redux state instead of explicit links?